cellbase-lib/src/main/java/org/opencb/cellbase/lib/impl/core/VariantMongoDBAdaptor.java

```diff
@@ -283,11 +305,11 @@ public Bson parseQuery(VariantQuery query) {
                 }
             }
         } catch (IllegalAccessException e) {
-            e.printStackTrace();
+            throw new CellBaseException(e.getMessage());
```

Comment:
By wrapping the exception with just the "e.getMessage", you're hiding the stacktrace, as the thrown exception won't have a "cause". 
It's always preferrable to have the original exception as cause, to have the full stacktrace.

The class "CellbaseException" doesn't have a constructor with "just an exception" as it is encouraging you to provide a more detailed exception of the problem. You could always be lazy, and just replicate the message. But always propagate the cause.

Bad:
```java
new CellBaseException(e.getMessage());
```

Good:
```java
new CellBaseException(e.getMessage(), e);
```

Better
```java
new CellBaseException("Error parsing variant query: " + Objects.toString(variantQuery), e);
```

Reply:
Ok, I'll use the option "better"